crates/engine/src/mock_rocksdb_engine.rs

```diff
@@ -197,3 +202,18 @@ impl SnapshotApi for RocksSnapshot {
         self.inner.clean().await
     }
 }
+
+/// A transaction of the `RocksEngine`
+#[derive(Copy, Clone, Debug, Default)]
+pub struct RocksTransaction;
```

Comment:
Why is this `RocksTransaction` empty?

Reply:
It's not implement yet, I'll implement it in another PR.